dev-tools/iota-network/genesis/overlays/common.yaml

```diff
@@ -2,7 +2,7 @@ validator:
   network-address: /ip4/0.0.0.0/tcp/8080/http
   metrics-address: 0.0.0.0:9184
   json-rpc-address: 0.0.0.0:9000
-  admin-interface-port: 1337
+  admin-interface-address: 127.0.0.1:1337
```

Comment:
No quotes here but quotes are in the `fullnode.yml`

Reply:
I went through all the files and added quotes to strings that have a colon.